NodeFirefox/Dockerfile

```diff
@@ -1,15 +1,21 @@
-FROM selenium/node-base:2.52.0
+FROM selenium/node-base:latest
 MAINTAINER Selenium <selenium-developers@googlegroups.com>
 
 USER root
 
 #=========
 # Firefox
 #=========
+ENV FIREFOX_VERSION 45.0.1
 RUN apt-get update -qqy \
-  && apt-get -qqy --no-install-recommends install \
-    firefox \
-  && rm -rf /var/lib/apt/lists/*
+  && apt-get -qqy --no-install-recommends install firefox \
+  && rm -rf /var/lib/apt/lists/* \
+  && wget --no-verbose -O /tmp/firefox.tar.bz2 https://download-installer.cdn.mozilla.net/pub/firefox/releases/$FIREFOX_VERSION/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.tar.bz2 \
+  && rm -rf /opt/firefox \
```

Comment:
Before the call to ```rm```: Add a line for deinstalling the packaged version of Firefox.
```
apt-get -y purge firefox
```